src/wallet/rpcdump.cpp

```diff
@@ -609,13 +609,27 @@ UniValue dumpwallet_impl(const UniValue& params, bool fDumpZKeys)
     file << strprintf("# * Created on %s\n", EncodeDumpTime(GetTime()));
     file << strprintf("# * Best block at time of backup was %i (%s),\n", chainActive.Height(), chainActive.Tip()->GetBlockHash().ToString());
     file << strprintf("#   mined on %s\n", EncodeDumpTime(chainActive.Tip()->GetBlockTime()));
-    {
-        HDSeed hdSeed;
-        pwalletMain->GetHDSeed(hdSeed);
-        auto rawSeed = hdSeed.RawSeed();
-        file << strprintf("# HDSeed=%s fingerprint=%s", HexStr(rawSeed.begin(), rawSeed.end()), hdSeed.Fingerprint().GetHex());
-        file << "\n";
+
+    std::optional<MnemonicSeed> hdSeed = pwalletMain->GetMnemonicSeed();
+    if (hdSeed.has_value()) {
+        auto mSeed = hdSeed.value();
+        file << strprintf(
+                "# Emergency Recovery Phrase=\"%s\" \n# language=%s \n# fingerprint=%s\n",
+                mSeed.GetMnemonic(),
+                MnemonicSeed::LanguageName(mSeed.GetLanguage()),
+                mSeed.Fingerprint().GetHex()
+                );
```

Comment:
We can't split on spaces to parse, because the recovery phrase itself includes spaces. That's why this is split over multiple lines. But I can change it to the dictionary-like format, that works.